scripts/install_third_party.py

```diff
@@ -140,162 +142,95 @@ def download_and_untar_files(
             os.path.join(target_parent_dir, target_root_name))
 
 
+def return_json(source_url):
+    """Return json object when provided url
+    Args:
+        source_url: the URL of the json file.
+    Return:
+        a parsed json objects
+    """
+    response = urllib.urlopen(source_url)
```

Comment:
I don't understand -- why is this a URL at all? Shouldn't it be a filepath? In which case I'd expect the logic to be similar to utils.get_file_contents...